docs/user/trusted-publishers/using-a-publisher.md

```diff
@@ -270,7 +270,47 @@ below describe the setup process for each supported trusted publisher.
 
 === "ActiveState"
 
-    TODO
+    ActiveState's Platform works as a "zero-config" CI solution to automatically build cross-platform wheels of your PyPI projects. Once you're set up on the Platform and have linked your PyPI project, you're ready to publish. For more information on getting started with ActiveState go [here](https://docs.activestate.com/platform/start/PYPI). To begin:
```

Comment:
Is there something we can link to that details the supported platforms/architectures/etc.?

Reply:
Good call. It's available via the API, but we'll look at putting something together in the linked docs that's easy to reference.